src/ontology/mondo-edit.obo

```diff
@@ -308137,7 +308140,7 @@ is_a: MONDO:0003778 {source="PMID:11941303", source="https://orcid.org/0000-0001
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0002-5002-8648"} ! hereditary disease
 is_a: MONDO:0005046 {source="https://orcid.org/0000-0001-8216-5084"} ! immune system disorder
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0013576 {source="MONDO:CLINGEN"}
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5716 {source="MONDO:mim2gene_medgen", source="OMIM:614102"} ! IGKC
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5716 {source="MONDO:mim2gene_medgen"} ! IGKC
```

Comment:
Can you confirm that this is correct? I see this annotation in OMIM, and in omim, it looks correct.

Reply:
It looks like `relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5716 {source="MONDO:mim2gene_medgen", source="OMIM:614102"} ! IGKC` should have been maintained.